init-tools.ps1

```diff
@@ -57,6 +57,61 @@ if (!(Test-Path "$BUILD_TOOLS_PACKAGE_PATH\init-tools.cmd"))
     exit 1
 }
 
+# Bring down the CLI for build tools
+$DOTNET_PATH=$BUILD_TOOLS_PATH + "\dotnetcli"
+
+Write-Host "Installing Build Tools CLI version..."
+if (!(Test-Path "$DOTNET_PATH"))
+{
+     mkdir "$DOTNET_PATH"
+}
+
+$DOTNET_VERSION = Get-Content "$RepoRoot\BuildToolsCLIVersion.txt"
+$DOTNET_ZIP_NAME="dotnet-dev-win-x64." + $DOTNET_VERSION + ".zip"
```

Comment:
This is due to incompatibility with pre-preview3 versions in the rel/1.0.0 version of the install script. 

I recommend doing what the release branch of core-setup does, downloading the install script from the preview2 branch, and running that. It's a workaround but is cleaner.